helm/credentiald-chart/templates/deployment.yaml

```diff
@@ -36,18 +36,6 @@ spec:
           items:
           - key: config.yml
             path: config.yml
-      affinity:
-        podAntiAffinity:
-          preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Just verifying: You are deleting this because it's invalid (wrong label `k8s-app`) and the right one is already defined above with `app` label?

Reply:
Yes. The `affinity` field is duplicated, and I figured the one I removed wasn't really working anyway.